pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_runtime_reconcile.go

```diff
@@ -73,6 +73,21 @@ func (c *runtimeReconcile) reconcile(ctx context.Context, cm *operatorapiv1.Clus
 		}
 	}
 
+	var enabledRegistrationDrivers []string
+	if cm.Spec.RegistrationConfiguration != nil {
+		for _, registrationDriver := range cm.Spec.RegistrationConfiguration.RegistrationDrivers {
+			enabledRegistrationDrivers = append(enabledRegistrationDrivers, registrationDriver.AuthType)
+			if registrationDriver.AuthType == "awsirsa" {
+				config.HubClusterArn = registrationDriver.HubClusterArn
+			}
+		}
+		// If no registration drivers are enabled, default to csr for backward compatibility
+		if len(enabledRegistrationDrivers) == 0 {
```

Comment:
since the default flag as csr as the default driver in registration controller already, we  do not to set default here. And it is better not to set new flag here for backward compatible.

Reply:
Addressed